helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -576,8 +573,8 @@ public void handleDataChange(String dataPath, Object data) {
 
   @Override
   public void handleChildChange(String parentPath, List<String> currentChilds) {
-    logger.debug("Data change callback: child changed, path: " + parentPath + ", current childs: "
-        + currentChilds);
+    logger.info("Data change callback: child changed, path: " + parentPath + ", current child count: "
```

Comment:
lets avoid constructing strings to reduce GC pressure. Also, this will pollute the logs. What are we trying to get out of these log messages

Reply:
Sure, I will change it to debug, we use that information to investigate our occusion callback lost issue, and now we know the cause, so will fix this.